subsys/usb/usb_transfer.c

```diff
@@ -61,11 +61,7 @@ bool usb_transfer_is_busy(uint8_t ep)
 {
 	struct usb_transfer_data *trans = usb_ep_get_transfer(ep);
 
-	if (trans && trans->status == -EBUSY) {
-		return true;
-	}
-
-	return false;
+	return trans && trans->status == -EBUSY;
```

Comment:
I do not see the reason to change it. AFAIK in C the type of logical operators is integer and therefore the existing code you replace is more correct.